src/Twilio/Base/Page.cs

```diff
@@ -78,9 +69,9 @@ private static string UrlFromUri(Domain domain, string region, string uri)
         /// <param name="domain">Twilio subdomain</param>
         /// <param name="region">Twilio region</param>
         /// <returns>URL for the first page of results</returns>
-        public string GetFirstPageUrl(Domain domain, string region)
+        public string GetFirstPageUrl(Domain domain, string region = null)
```

Comment:
is this just for backwards compatibility? Don't see region being used in this func.

Reply:
Yes, it will be dropped soon but requires a yoyo change.